pghoard/rohmu/object_storage/sftp.py

```diff
@@ -0,0 +1,209 @@
+"""
+rohmu - sftp object store interface
+
+Copyright (c) 2016 Ohmu Ltd
+See LICENSE for details
+"""
+
+import logging
+import warnings
+import pysftp
+from pysftp import CnOpts
+import os
+import datetime
+from stat import S_ISDIR
+from io import BytesIO, StringIO
+import json
```

Comment:
The import ordering doesn't seem to be all that consistent in existing code either but mostly it would seem to be `from` imports first followed by `import` imports, both sorted alphabetically within the group. That would look nicer than the current mix of imports used here.